<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>章节目录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>book2.png</normaloff>book2.png</iconset>
  </property>
  <widget class="QTextBrowser" name="tb_dir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
